TransferMethod/Sources/AddTransferMethodController.swift

```diff
@@ -38,13 +38,11 @@ public final class AddTransferMethodController: UITableViewController {
     }()
 
     // MARK: - Properties -
-    /// The completion handler will be performed after a new transfer method has been created.
-    public var createTransferMethodHandler: ((HyperwalletTransferMethod) -> Void)?
-    private var country: String
-    private var currency: String
-    private var forceUpdate: Bool
-    private var profileType: String
-    private var transferMethodTypeCode: String
+    private var country: String?
+    private var currency: String?
+    private var forceUpdate: Bool?
```

Comment:
forceUpdate is not used anywhere so remove it

Reply:
Removed forceUpdate